client/my-sites/stats/activity-log-item/activity-actor.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default class ActivityActor extends PureComponent {
+	static propTypes = {
+		avatarUrl: PropTypes.string.isRequired,
+		displayName: PropTypes.string,
+		login: PropTypes.string,
+		translatedRole: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		avatarUrl: 'https://www.gravatar.com/avatar/0',
```

Comment:
My understanding:

* `isRequired` -> this component will act assuming this prop is set
* in `defaultProps` -> guarantee the prop is set

I think often required props are the _best_ candidates for defaults.